step.yml

```diff
@@ -1,13 +1,26 @@
 title: "Change Working Directory for subsequent Steps"
-summary: |-
-  Set bitrise source directory (`BITRISE_SOURCE_DIR`) to a specified path.
+summary: Changes Bitrise source directory (`BITRISE_SOURCE_DIR`) to a specified path.
 description: |-
+  This Step changes Bitrise source directory (`BITRISE_SOURCE_DIR`) to a specified path.
+
   By setting `$BITRISE_SOURCE_DIR` to the specified path every subsequent
   Step will run with this path as its base Working Directory.
```

Comment:
I'd remove this part and add the following sentence right after the first sentence to avoid some repetition. Wdyt?
```
This will cause every subsequent Step to run with this path as its base Working Directory.
```

Reply:
Good!